config/resource-manager.hcl

```
@@ -151,7 +151,7 @@ service "datamigration" {
 }
 service "dataprotection" {
   name      = "DataProtection"
-  available = ["2022-04-01", "2022-05-01", "2022-12-01", "2023-01-01", "2023-05-01"]
+  available = ["2022-04-01", "2022-05-01", "2022-12-01", "2023-01-01"]
```

we'll need to do:

```suggestion
  available = ["2022-04-01", "2022-05-01", "2022-12-01", "2023-01-01"]
  ignore = ["2023-05-01"]
```

else this'll get re-added in the next config update